src/Facades/Screen.php

```diff
@@ -7,6 +7,7 @@
 /**
  * @method static object cursorPosition()
  * @method static array displays()
+ * @method static array getCenterOfActiveScreen()
```

Comment:
I forgot to add the `active` and `primary` method signatures here - could you do it?

Reply:
yes